spec/dummy/config/environments/test.rb

```diff
@@ -10,6 +10,6 @@
   config.action_controller.allow_forgery_protection = false
   config.action_mailer.default_url_options = { host: 'localhost:3000' }
   config.action_mailer.delivery_method = :test
-  config.action_mailer.perform_deliveries = false
+  config.action_mailer.perform_deliveries = true
```

Comment:
Was this causing emails not to populate the ActionMailer emails array during tests? Eesh that's a rough one to have been missing all this time